src/promptflow-evals/promptflow/evals/evaluate/_batch_run_client/batch_run_context.py

```diff
@@ -25,10 +26,16 @@ def __enter__(self):
             os.environ[PF_FLOW_ENTRY_IN_TMP] = "true"
             os.environ[PF_FLOW_META_LOAD_IN_SUBPROCESS] = "false"
 
+            if os.environ.get(PF_BATCH_TIMEOUT_SEC) is None:
```

Comment:
Any specific need for setting it if customer does not set it ?

Reply:
If they haven't set it, we set it to the default. If they have set it, we leave it alone.